docs/hub-tutorials/join-mainnet.md

```diff
@@ -215,16 +189,169 @@ enable = true
 address = "0.0.0.0:9090"
 ```
 
-## Upgrades
 
-To be best prepared for eventual upgrades, it is recommended to setup [Cosmovisor](https://docs.cosmos.network/master/run-node/cosmovisor.html), a small process manager,  which can swap in new `gaiad` binaries. Read more about setting this up [here](upgrade-node.md).
+## Choosing a Sync Mode
+
+There are three main ways to sync a node on the Cosmos Hub. The way a node operator will want to sync is partially dependent on the type of node they will operate. This guide will focus on two types of common nodes; archive/full and pruned. For further information on syncing to run a validator node, see the section on [Validators](https://hub.cosmos.network/main/validators/overview.html).
 
-## Background Process
+If a node operator wishes to run a full/archive node, it is possible to start from scratch but will take a significant amount of time to catch up. Node operators not concerned with rebuilding original state from the beginning of `cosmoshub-4` can also leverage [Quicksync](#Quicksync)'s available archive history.
 
-To run the node in a background process with automatic restarts, you can use a service manager like `systemd`. To set this up run the following:
+For operators interested in bootstrapping a pruned node, either [Quicksync](#Quicksync) or [State Sync](#State-Sync) would be sufficient.
 
+Make sure to consult the [hardware](#Hardware) section for guidance on the best configuration for the type of node operating.
+
+### Sync from scratch
+
+Syncing from scratch is possible by starting the chain from the beginning of `cosmoshub-4` and either manually upgrading the chain or setting up [Cosmovisor](#Cosmovisor) to upgrade automatically. Node operators syncing from scratch will need to navigate two upgrades, Delta and Vega. For more information on performing the manual upgrades, see [Releases & Upgrades](#Releases-amp=-Upgrades).
```

Comment:
Small suggestion to make it more clear users cannot sync from scratch with v6.

```suggestion
Syncing from scratch is possible by starting the chain from the beginning of `cosmoshub-4` and either manually upgrading the binary or setting up [Cosmovisor](#Cosmovisor) to upgrade automatically. Node operators syncing from scratch will need to navigate two upgrades, Delta and Vega. For more information on performing the manual upgrades, see [Releases & Upgrades](#Releases-amp=-Upgrades). Make sure you have installed Gaia [`V4.2.1`](https://github.com/cosmos/gaia/releases/tag/v4.2.1).
```

Reply:
```suggestion
Syncing from scratch is possible by starting the chain from the beginning of `cosmoshub-4` and either manually upgrading the chain or setting up [Cosmovisor](#Cosmovisor) to upgrade automatically. Node operators syncing from scratch will need to navigate two upgrades, Delta (from [gaia v4.2.1](https://github.com/cosmos/gaia/releases/tag/v4.2.1) to [gaia v5.0.8](https://github.com/cosmos/gaia/releases/tag/v5.0.8)) and Vega (from [gaia v5.0.8](https://github.com/cosmos/gaia/releases/tag/v5.0.8) to [gaia v6.0.0](https://github.com/cosmos/gaia/releases/tag/v6.0.0)). For more information on performing the manual upgrades, see [Releases & Upgrades](#Releases-amp=-Upgrades).
```